contributors/devel/release/issues.md

```diff
@@ -0,0 +1,193 @@
+    2. it has not been completed and should remain open
+    3. it is being actively worked on by the assignee
+    
+    **Note**: This issue must be resolved or moved out of the milestone by <date of feature freeze +2 weeks>
+    ```
+    - after 2 days without a reply, bot follows escalation procedure derived from priority label
+      - important-longterm, remove from milestone and notify the owners
+      - important-soon, escalate to SIG daily for 2 days.
+    - removal should add the `release/remove-from-milestone` label and contain the following message
+      ```
+      **Important**:
+      There has been no confirmation that this issue belongs in the v1.X milestone.
+      Removing it from the milestone.
+      ```
+  - **priority/critical-urgent issues**
+    - bot comments on issues not updated within 3 days mentioning issue owners, expects updates every 3 days going forward.
+      ```
+      **Action Required**:
+      This issue is marked as priority/critical-urgent, but has not been updated in 3 days.  Please provide
+      an update or the issue will be escalated to the SIG.
+      ```
+    - owner updates can be a short ACK, but should include an ETA for completion and any risk factors.
+      ```
+      ACK.  In progress
+      ETA: DD/MM/YYYY
+      Risks: Complicate fix required
+      ```
+      
+      ```
+      ACK.  In progress
+      ETA: ???
+      Risks: Root cause unknown.
+      ```
+    - `release/needs-attention`
+4. 2 weeks after feature freeze
+  - **priority/important-* issues**
+    - bot comments mentioning issue owners, expects the issue to be escalated in priority or removed from milestone
+    ```
+    **Action Required**:
+    This issue is marked as priority/important-x, and is in the 1.y milestone.  Only critical-urgent issue
+    are being tracked for the 1.y release.  Escalate the priority to critical-urgent immediately, **or the issue
+    will be removed from the milestone**.
+    ```
+    - bot immediately escalates to the SIG.  escalation message includes link to all priority/important issues
+      for the SIG remaining in the milestone.
+  - **priority/critical-urgent issues**
+    - bot comments on issues not updated within 1 day mentioning issue owners, expects updates every day going forward.
+    - bot escalates to SIG after 2 days without updates and applies the label `release/needs-attention`
+
+## Escalation
+
+SIGs will have issues needing attention escalated through the following channels
+
+- Comment mentioning the sig team appropriate for the issue type
+- Email the SIG googlegroup list
+  - maybe configured to email alternate googlegroup
+  - maybe configured to directly email SIG leads or other SIG members
+- Message the SIG slack channel, mentioning the SIG leads
+  - maybe configured to message alternate slack channel and users
+
+## Issues tracked in other repos
+
+Some issues are filed in repos outside the [kubernetes repo].  For these repo's issues
```

Comment:
Or this rather: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+milestone%3Av1.7+label%3Asig%2Fcli+org%3Akubernetes